VotingApplication/VotingApplication.Web.Api/Controllers/Web Controllers/AdminController.cs

```diff
@@ -1,13 +1,13 @@
-﻿using System.Web.Mvc;
+﻿using System;
+using System.Web.Mvc;
 
-namespace VotingApplication.Web.Api.Controllers.Web_Controllers
+namespace VotingApplication.Web.Api.Controllers
```

Comment:
If we're changing the namespace, we should change the directory too (as any new items added in will have the namespace derived from the path)